beets/autotag/match.py

```diff
@@ -179,33 +177,31 @@ def _recommendation(
     return rec
 
 
-AnyMatch = TypeVar("AnyMatch", TrackMatch, AlbumMatch)
-
-
 def _sort_candidates(candidates: Iterable[AnyMatch]) -> Sequence[AnyMatch]:
     """Sort candidates by distance."""
     return sorted(candidates, key=lambda match: match.distance)
 
 
 def _add_candidate(
-    items: Sequence[Item],
-    results: dict[Any, AlbumMatch],
-    info: AlbumInfo,
+    items: Sequence[Item], results: Candidates[AlbumMatch], info: AlbumInfo
```

Comment:
Can we try to keep the trailing comma inplace if it exists already. This makes the git diff easier to read.